tests/common.py

```diff
@@ -45,7 +46,10 @@
 _pd_flights = pd.DataFrame.from_records(flight_records).reindex(
     _ed_flights.columns, axis=1
 )
-_pd_flights["timestamp"] = pd.to_datetime(_pd_flights["timestamp"])
+if PANDAS_VERSION[0] == 2:
```

Comment:
To prepare for pandas 3

```suggestion
if PANDAS_VERSION[0] >= 2:
```